frontend/src/components/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableRow from "@mui/material/TableRow";
+import TableCell from "@mui/material/TableCell";
+
+interface Props {
+  title: string;
+  description: string;
+  table: {
+    item: string;
+    value: string;
+    link: string;
+  }[];
+}
+
+function InfoCard({ title, description, table }: Props) {
+  return (
+    <Card sx={{minWidth:723, maxWidth: 1023}}>
+      <CardContent>
```

Comment:
```suggestion
    <Paper variant="outlined"  sx={{minWidth:723, maxWidth: 1023}}>
```
クリック要素が入らないのでPaperがよき

Reply:
InfoCard自体にクリック要素はないので

- Paperコンポーネント使用
- 影はなし

でお願いします

Reply:
ここがまだ対応されてないかも？
variant="outlined"
がおすすめです